src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/config/DiffProperties.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.prisonersearch.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import org.springframework.boot.context.properties.ConstructorBinding
+
+@ConstructorBinding
+@ConfigurationProperties(prefix = "diff")
+data class DiffProperties(val telemetry: Boolean)
```

Comment:
Doesn't this boolean need to be nullable - can't see the default of `false` in the `values.yaml` file?

Reply:
I put the default in `application.yml`.

Reply:
just spotted the default in `application.yaml`, which explains what is going on - so ignore me